schema.graphql

```diff
@@ -25,16 +27,23 @@ type DomainTokenSold @entity {
 
 type BuyNowListing @entity {
   id: String! # tokenId
+  indexId: Int!
   amount: BigInt!
   paymentToken: String
 }
 
 type Cancellation @entity {
   id: ID!
+  indexId: Int!
   bidNonce: BigInt!
   bidder: Account!
 }
 
 type Account @entity {
   id: ID!
 }
+
+type Global @entity {
+  id: ID!
+  uniqueEntityId: Int!
```

Comment:
what's this global type definition do?  Is it storing some state during a query?

Reply:
the max value for `skip` in a query is 5000, and the max value for `count` is 1000 so having a custom index property allows us to do [better pagination](https://thegraph.com/docs/en/developer/graphql-api/#pagination) once there are more than 6000 entities.